Cargo.toml

```diff
@@ -57,6 +57,7 @@ uint = "0.9.5"
 
 [dev-dependencies]
 tempfile = "3.8.1"
+casper-types = { version = "5.0.0", features = ["std", "json-schema", "testing"] }
```

Comment:
Shouldn't we be pointing to `feat-2.0` ?

Reply:
Yes, it's configured below.